runtime/near-vm-runner/src/wasmtime_runner.rs

```diff
@@ -285,8 +304,15 @@ impl crate::runner::VM for WasmtimeVM {
                     let err = VMError::FunctionCallError(FunctionCallError::MethodResolveError(
                         MethodResolveError::MethodNotFound,
                     ));
-                    // TODO: This should return an outcome to account for loading cost
-                    VMResult::nop_outcome(err)
+                    if checked_feature!(
+                        "protocol_feature_fix_contract_loading_cost",
+                        FixContractLoadingCost,
+                        current_protocol_version
+                    ) {
+                        return VMResult::abort(logic, err);
+                    } else {
+                        return VMResult::nop_outcome(err);
```

Comment:
Are there any instances where `nop_outcome` can be called with this feature enabled? If not, maybe it would make sense to add an `assert` in it?

Reply:
Yes, there are several places we still call `nop_outcome`. Perhaps the most important location is in `runtime/runtime/src/actions.rs` in the body of `execute_function_call()`.

https://github.com/near/nearcore/blob/0775622fc2cd6cd428960b4cd10867e50fef90a9/runtime/runtime/src/actions.rs#L39-L67

Hopefully, I will eventually remove all of these.